client/rest-high-level/src/test/java/org/elasticsearch/client/ESRestHighLevelClientTestCase.java

```diff
@@ -134,6 +134,7 @@ protected static void clusterUpdateSettings(Settings persistentSettings,
         ClusterUpdateSettingsRequest request = new ClusterUpdateSettingsRequest();
         request.persistentSettings(persistentSettings);
         request.transientSettings(transientSettings);
-        assertOK(client().performRequest(RequestConverters.clusterPutSettings(request)));
+        assertTrue(execute(
```

Comment:
I moved to using the HLRC here because I did not want the ClusterRequestConverter used in this class.

Reply:
I *think* it should be enough to just call the API without any assertion. We already should already throw an exception if the response isn't a 200.

Reply:
heh, so then we should remove the bool on the `AcknowledgedResponse` :)